src/pages/identity/IdentityDetails.js

```diff
@@ -43,28 +55,63 @@ class IdentityDetails extends Component {
                         { address &&
                             <IdentityAttribute title="Location" value={ address } /> }
                     </div>
-                    <ModalTrigger modal={ <EditProfile id={ id } /> }>
-                        <Button variant="negative" className={ styles.editButton } onClick={ this.handleEdit }>
-                            Edit<span className={ styles.profileSpan }> Profile</span> <EditIcon className={ styles.buttonIcon } />
-                        </Button>
-                    </ModalTrigger>
+                    <SplitButton
+                        variant="negative"
+                        actions={ actions }
+                        mainActionId={ mainActionId }
+                        onActionClick={ this.handleActionClick } />
                 </div>
             </div>
         );
     }
 
-    handleEdit = () => {
-        alert('Edit profile');
```

Comment:
Can’t we use memoizeOne directly here?

Reply:
Yes, we can, but then eslint will ask us to place this code before the constructor. What should we do?

Reply:
Damn, we really need to fix that eslint rule. Is the order rule right?

Reply:
Perhaps we can drop a eslint ignore for now?

Reply:
Sure! I need to change this kind of stuff in some other components.